### DISCUSSION TITLE
Helix equivelant for vi" in neovim

### DISCUSSION THREAD

**Discussion by x0ba** [helix-editor/helix#7847] [Q&A]
is there a helix equivelant of something like vi" in vim? where you can select everything inside the double quotes and jump to the next double quotes if you're not inside double quotes already? mi" exists but it doesnt have the aforementioned jumping feature.

**Comment by ghost** [comment]
I am still learning, but I hope this helps.

You can mimic this functionality with search and cycling through the hits. To select all text inside a pair, use f / t.

`:/"` : searches for " inside your active buffer.
`n` / `N` : move to the next / previous match for your latest search in your active buffer.
`f"`  / `F"` : select everything from the current cursor position to the next / previous instance of the double quote char.

`mi"`: You can select all text inside a string with this motion.
`ma"`: You can select all text inside a string including the quotes with this motion.
`mm`: You can jump to the matching " with your cursor on " or inside the string.
